docs/atomic-run.1.md

```diff
@@ -49,6 +49,9 @@ These defaults are suggested values for your container images.
 **SUDO_GID**
   The `SUDO_GID` environment variable.  This is useful with the docker `-u` option for user space tools.  If the environment variable is not available, the default GID of the value for `SUDO_UID` is used.  If this value is not available, the value of `/proc/self/loginuid` is used.
 
+**RUN_OPTS**
```

Comment:
OK, the LABEL can be OPTS_FILE. What should the "environment variable" in the `docker run` command instead of `${RUN_OPTS}` be called? `${OPTS}`? `${OPTS_FILE_CONTENT}`? Calling it just `${OPTS_FILE}` seems weird, as it won't be expanded to file name but to content of that file.